_ingest-pipelines/processors/html-strip.md

```diff
@@ -0,0 +1,169 @@
+---
+layout: default
+title: HTML strip
+parent: Ingest processors
+nav_order: 140
+---
+
+# HTML strip processor
+
+The `html_strip` processor removes HTML tags from string fields in incoming documents. This processor is useful when indexing data from webpages or other sources that may contain HTML markup. HTML tags are replaced with newline characters (`\n`).
+
+The following is the syntax for the `html_strip` processor:
+
+```json
+{  
+  "html_strip": {  
+    "field": "webpage"  
+  }  
+}  
+```
+{% include copy-curl.html %}
+
+## Configuration parameters
+
+The following table lists the required and optional parameters for the `html_strip` processor.
+
+Parameter | Required/Optional | Description |
+|-----------|-----------|-----------|
+`field` | Required | The string field from which to remove HTML tags.
+`target_field` | Optional | The field to assign the cleaned value to. If not specified, then the field is updated in-place.
```

Comment:
```suggestion
`target_field` | Optional | The field to receive the plain text version after stripping HTML tags. If not specified, then the field is updated in-place.
```

Reply:
@natebower Please see revision made to clarify "cleaned value."

Reply:
LGTM, but "to receive" => "that receives"?